jerry-core/ecma/base/ecma-globals.h

```diff
@@ -343,6 +343,38 @@ typedef enum
   ECMA_PROPERTY_TYPE__MAX = ECMA_PROPERTY_TYPE_VIRTUAL, /**< highest value for property types. */
 } ecma_property_types_t;
 
+/**
+ * Property name listing options.
+ */
+typedef enum
+{
+  ECMA_LIST_NO_OPTS = (0), /**< no options are provided */
+  ECMA_LIST_ARRAY_INDICES = (1 << 0), /**< exclude properties with names
+                                       *   that are not indices */
+  ECMA_LIST_ENUMERABLE = (1 << 1), /**< exclude non-enumerable properties */
+  ECMA_LIST_PROTOTYPE = (1 << 2), /**< list properties from prototype chain */
+} ecma_list_properties_options_t;
+
+/**
+ * List array indices and enumerable properties.
+ */
+#define ECMA_LIST_ARRAY_INDICES_ENUMERABLE (ECMA_LIST_ARRAY_INDICES | ECMA_LIST_ENUMERABLE)
```

Comment:
Some defines from this list is not used. Please remove them. We will introduce them if needed.

Reply:
I've removed them.